src/pages/registration/contributors_tab/components/AddContributorModalContent.tsx

```diff
@@ -128,7 +124,6 @@ export const AddContributorModalContent = ({
           variant="contained">
           {initialSearchTerm ? t('contributors.verify_person') : t('common:add')}
         </StyledVerifyButton>
-        <StyledCloseButton onClick={handleCloseModal}>{t('common:close')}</StyledCloseButton>
```

Comment:
hmm, ble veldig skeptisk først, men ser vi har ulike løsninger for dette overalt. kryss og knapp, bare kryss, bare knapp, så det er kasnkje ikke så farlig 🤷‍♂️ det er uansett ikke sånn at det mangler actions i bunnen av modalen der :p